src/index.html

```diff
@@ -6,9 +6,39 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          sizes="40px"
+          alt="logo">
+        <nav class="navbar">
+          <a href="#" class="is-active nav__link">APPLE</a>
+          <a href="#" class="nav__link">SAMSUNG</a>
+          <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          <a href="#"
+            class="nav__link comp"
+            data-qa="hover">
+            НОУТБУКИ И КОМПЬЮТЕРЫ
+          </a>
```

Comment:
```suggestion
          <a href="#"
            class="nav__link comp"
            data-qa="hover"
          >
            ноутбуки и компьютеры
          </a>
```
Пиши назви у нижньому регістрі, і роби букви великими у css (text-transform)